index.html

```diff
@@ -36,6 +36,196 @@ <h1>
             <li><a href="#"><img src="./assets/victory.svg" alt="victory"></a></li>
         </ul>
     </section>
+
+    <section id="works">
+        <div class="project">
+            <div>
+                <img src="./assets/SnapshootPortfolio.svg" alt="">
+            </div>
+
+            <div class="projectDescription">
+                <div class="projectTitle">
+                    <h1>Tonic</h1>
+                </div>
+                <ul class="projectTools">
+                    <li>CANOPY</li>
+                    <li><img src="./assets/Counter.svg" alt=""></li>
+                    <li>Back End Dev</li>
+                    <li><img src="./assets/Counter.svg" alt=""></li>
+                    <li>2015</li>
+                </ul>
+                <div class="projectSummary">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="projectTechnology">
+                    <span class="tag">HTML</span>
+                    <span class="tag">CSS</span>
+                    <span class="tag">JavaScript</span>
+                </div>
+                <div class="projectDemo">
+                    <button type="button">See project</button>
+                </div>
+            </div>
+        </div>
+
+        <div class="project">
+            <div>
+                <img src="./assets/SnapshootPortfolio1.svg" alt="">
+            </div>
+
+            <div class="projectDescription">
+                <div class="projectTitle">
+                    <h1>Multi-Post
+                        Stories</h1>
+                </div>
+                <ul class="projectTools">
+                    <li>CANOPY</li>
+                    <li><img src="./assets/Counter.svg" alt=""></li>
+                    <li>Back End Dev</li>
+                    <li><img src="./assets/Counter.svg" alt=""></li>
+                    <li>2015</li>
```

Comment:
- [ ] Please according to the requirement, You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using this [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=0-1&t=tlnrHuarxypt0Dez-0). Below screenshot is the difference between the given design and yours: :arrow_down: 

<table>
    <tr>
      <td>Your design</td>
      <td>Figma design</td>
    </tr>
    <tr>
      <td>

![Your Canopy](https://github.com/BilalLiaquat7/Portfolio/assets/43528594/555ddb00-35a2-4853-992a-29de2a41736a)

</td>
      <td>

![Canopy D figma](https://github.com/BilalLiaquat7/Portfolio/assets/43528594/c080dd2c-5187-41ea-a845-f2639aa7cca6)

</td>
    </tr>
  </table>